scripts/update-cookie-db.cjs

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This script is used to format the data from the open-cookie-database.csv file and write it back to open-cookie-database.json file.
+ */
+const fs = require('fs-extra');
+const path = require('path');
+const csvtojson = require('csvtojson');
+const process = require('process');
+const fetch = require('node-fetch');
+const targetDIR = path.resolve(__dirname, '..', 'third_party/data');
+
+// @todo Update to https://raw.githubusercontent.com/jkwakman/Open-Cookie-Database/master/open-cookie-database.csv after
+// https://github.com/jkwakman/Open-Cookie-Database/pull/32 is merged.
+const requestURL =
+  'https://raw.githubusercontent.com/rtCamp/Open-Cookie-Database/add-wordpress-cookies/open-cookie-database.csv';
```

Comment:
@ayushnirwal Temporarily updated the url to fetch the updated cookie database until https://github.com/jkwakman/Open-Cookie-Database/pull/32 is merged